tests/main/security-device-cgroups/task.yaml

```diff
@@ -0,0 +1,59 @@
+summary: Ensure that the security rules related to device cgroups work.
+
+environment:
+    DEVICE_NAME/kmsg: kmsg
+    UDEVADM_PATH/kmsg: /sys/devices/virtual/mem/kmsg
+    DEVICE_ID/kmsg: "c 1:11 rwm"
+    OTHER_DEVICE_NAME/kmsg: uinput
+    OTHER_UDEVADM_PATH/kmsg: /sys/devices/virtual/misc/uinput
+    OTHER_DEVICE_ID/kmsg: "c 10:223 rwm"
+
+    DEVICE_NAME/uinput: uinput
+    UDEVADM_PATH/uinput: /sys/devices/virtual/misc/uinput
+    DEVICE_ID/uinput: "c 10:223 rwm"
+    OTHER_DEVICE_NAME/uinput: kmsg
+    OTHER_UDEVADM_PATH/uinput: /sys/devices/virtual/mem/kmsg
+    OTHER_DEVICE_ID/uinput: "c 1:11 rwm"
+
+restore: |
+    rm -f test-snapd-tools_1.0_all.snap /etc/udev/rules.d/70-snap.test-snapd-tools.rules
+    udevadm control --reload-rules
+    udevadm trigger
+
+execute: |
+    echo "Given a snap is installed"
+    snapbuild $TESTSLIB/snaps/test-snapd-tools .
+    snap install ./test-snapd-tools_1.0_all.snap
+
+    echo "Then the device is not assigned to that snap"
+    ! udevadm info $UDEVADM_PATH | grep -Pq "E: TAGS=.*?snap_test-snapd-tools_env"
```

Comment:
It isn't clear to me how $UDEVADM_PATH here maps to UDEVADM_PATH/kmsg and UDEVADM_PATH/uinput. Can you comment?